components/BottomMenu.js

```diff
@@ -76,8 +75,12 @@ function BottomMenu(props) {
     };
 
     const goToNearby = () => {
-        props.navigation.navigate("Nearby");
-    };
+        props.navigation.navigate("Nearby", {bottomMenu: true});
+    }
+
+    const currentAltitude = async () => {
+        let altitude = await AsyncStorage.getItem('altitude');
+    }
```

Comment:
<img width="754" alt="Screen Shot 2020-04-01 at 4 03 01 PM" src="https://user-images.githubusercontent.com/35850933/78181386-4602d900-7432-11ea-8d26-8bd8122469bb.png">

Reply:
This was already on the master branch! I can check it

Reply:
oh! for some reason sonar indicated that it came with this PR. Forget it if you didn't work on this part.